checklists/alz_checklist.en.json

```diff
@@ -2105,6 +2105,17 @@
             "training": "https://azure.github.io/azure-monitor-baseline-alerts/patterns/alz/",
             "link": "https://learn.microsoft.com/azure/cloud-adoption-framework/ready/landing-zone/design-area/management-monitor"
         },
+        {
+            "category": "Management",
+            "subcategory": "Monitoring",
+            "text": "Use Azure Monitoring Agent (AMA). The Log Analytics agent is on a deprecation path and won't be supported after August 31,2024",
```

Comment:
Do we want to say that "it is deprecated" instead of being on the deprecation path?  Since it is just a few weeks left.

Reply:
I will make the change; thank you for the suggestion